### PR TITLE
Add PR's to the changelog for 2.5.0.0.4

### PR DESCRIPTION
**Pull request by wileyj**
Changelog updates for 2.5.0.0.4

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 - Update /v2/fees/transfer to report the median transaction fee estimate for a STX-transfer of 180 bytes (#4754)
 - Reduce connection spamming in stackerdb (#4759)
 - Remove deprecated signer cli commands (#4772)
+- Fix/signer-slot-count (#4868)
+- Fix: slots count in the .signers contract (#4891)
```
[the remark this answers is not in the document]
Comment (wileyj) [reply CHANGELOG.md:36]: i'm not entirely sure - they both showed up in the merge history

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
...
+- Fix: never consider Stacks chain tips that are not on the canonical burn chain #4886 (#4893)
```
[the remark this answers is not in the document]
Comment (wileyj) [reply CHANGELOG.md]: udpated

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (wileyj) [reply CHANGELOG.md]: looks like 4891 builds on 4868. addressed in latest commit

[the remark this answers is not in the document]
Comment (obycode) [reply CHANGELOG.md]: I think it's okay to just have one line item: "Fix slots count in the .signers contract (#4845, #4868, #4891)"

**Review by obycode — APPROVED** [review]
👍

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (wileyj) [reply CHANGELOG.md]: exactly what i formatted it with

**Review by jcnelson — APPROVED** [review]

**Pull request merged**